lib/matplotlib/tests/test_simplification.py

```diff
@@ -13,11 +15,14 @@
 from matplotlib.path import Path
 
 
-# NOTE: All of these tests assume that path.simplify is set to True
-# (the default)
+@pytest.fixture
+def set_simplify_and_threshold():
```

Comment:
There is exactly one test that turns it off, but it's turned off manually in the test... so no, apart from me not reading pytest docs thoroughly :)